galleries/plot_types/unstructured/README.rst

```diff
@@ -1,9 +1,7 @@
 .. _unstructured_plots:
 
-Unstructured coordinates
--------------------------
+Irregularly gridded data: ``Z[i,j]``
```

Comment:
The index notation does not make sense here. The irregularity implies that you generally do not have two independent axes (i / j) that you can use to describe your data. For example, what should be the indexing scheme for the points `[(1, 0), (0, 1), (2, 2)]`?